src/definitions/pathRepresentation.ts

```diff
@@ -0,0 +1,11 @@
+import { int, Nullable } from "./utils";
+
+export type pathRepresentation = {
```

Comment:
With types like this you can/should do it PascalCase and maybe drop the representation part...: `Path`

PS: Maybe its best to move it into the definitions/placeTile.ts and rename that file to adjacency.ts or so